components/project/activity/index.js

```diff
@@ -0,0 +1,84 @@
+import * as prettyDate from "pretty-date";
+
+const _records = [
```

Comment:
@MaxLascombe let me know what the data from the canister will look like and when the function is ready. UI is ready

Reply:
I'll attempt to do the querying from frontend myself and then ill push what I have and you can have a look. It's going to be a bit hacky because I going to try and get it to work with the current escrow canister functions so that we have backwards compatibility.